templates/service-app/csharp/tizen-manifest.xml.tmpl

```diff
@@ -0,0 +1,14 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<manifest package="{{androidIdentifier}}" version="1.0.0" api-version="4.0" xmlns="http://tizen.org/ns/packages">
+    <profile name="common"/>
+    <service-application appid="{{androidIdentifier}}" exec="Runner.dll" type="dotnet" multiple="false" nodisplay="true" taskmanage="false" api-version="4" auto-restart="false">
+        <label>{{projectName}}</label>
+        <icon>ic_launcher.png</icon>
+        <background-category value="media"/>
+        <metadata key="http://tizen.org/metadata/prefer_dotnet_aot" value="true"/>
+    </service-application>
+    <tv-info api-version="8.8.0">
+        <infolink>T-INFOLINK2021-1000</infolink>
+    </tv-info>
```

Comment:
why do we need this information? I think it would be nice to provide a `--tizen-profile` flag for `create` command later.

Reply:
> why do we need this information?

The information is required by the TV seller portal: https://github.com/flutter-tizen/flutter-tizen/pull/139.

> I think it would be nice to provide a `--tizen-profile` flag for `create` command later. 

There could be many side effects made by this. I'll explain at your seat.